utils/installer/install.sh

```diff
@@ -100,6 +110,14 @@ installpacker() {
 }
 
 cloneconfig() {
+	if [ -d "/data/data/com.termux" ]; then
+		sudo() {
+			eval "$@"
+		}
+		bin_path="$HOME/../usr/bin"
```

Comment:
can you define this variable outside of the function
like this for example
```shell
declare USER_BIN_DIR="/usr/local/bin"
```
and then use and change it wherever you want